flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/attribute/AttributeJpaRepository.java

```diff
@@ -98,5 +98,5 @@ SELECT COUNT(a) > 0
               LEFT JOIN KitVersionJpaEntity kv ON a.subject.kitVersionId = kv.id
               WHERE  a.id = :id AND kv.kit.id = :kitId
         """)
-    boolean existsByIdAndKitId(@Param("id") long id, @Param("kitId") long kitId);
+    boolean existsByIdAndKitVersionId(@Param("id") long id, @Param("kitId") long kitId);
```

Comment:
Change `kitId` to `kitVersionId` in parameter list and in query

`WHERE  a.id = :id AND kv.id = :kitVersionId`